docs/multi-process-images.md

```diff
@@ -0,0 +1,112 @@
+# Multi-Process Images
+
+For a quick start guide on how to run and use these containers, see [Pulp in One Container](https://pulpproject.org/pulp-in-one-container/).
+
+For more detailed instructions, see the [PULP OCI Images
+docs](https://docs.pulpproject.org/pulp_oci_images/).
+
+## pulp
+
+A single [Pulp](https://github.com/pulp/pulpcore) image that runs Pulp, as well as its third-party
+services (nginx, postgresql and redis), in one single Docker/Podman container.
+
+To run all the services, you do not need to specify a container command ("CMD"). The default CMD is:
+
+- **/init** - The [s6 service manager](https://github.com/just-containers/s6-overlay) that runs all the services.
+
+The image can also function the same as the Single-Process image ["pulp-minimal"](#pulp-minimal). Specifically, the image can also be run as as each of the following individual services, specified as the container command ("CMD").
+
+- **pulp-api** - serves the Pulp(v3) API. The number of instances of this service should be scaled as demand requires.  _Administrators and users of all of the APIs put demand on this service_. If pulp_python or pulp_container are in use, _Content consumers also put demand on this service_.
+
+- **pulp-content** - serves content to clients. pulpcore-api redirects clients to pulpcore-content to download content. When content is being mirrored from a remote source, this service can download that content and stream it to the client the first time the content is requested. The number of instances of this service should be scaled as demand requires. _Content consumers put demand on this service_.
```

Comment:
pulpcore-api redirects clients to pulpcore-content in the case of pulp_ansible, pulp_python, and pulp_container. For all other plugins, the clients go directly to pulpcore-content.

Reply:
OK, I added pulp_ansible also.

I feel that the current wording is appropriate. I think engineers/admins mostly care about which set of users will access each service.